src/facts/purple-coneflower.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Purple Coneflower
+image:
+  src: ../images/15264929397_9db643db03_k.jpg
+  url: https://www.flickr.com/photos/kewing/15264929397/in/photolist-pfUKFT-pvmF13-f6RU7B-eyN3pP-eyRbEA-513ziF-75FaX-tw15Uh-pfUw6C-9XQKck-9RvayA-JCveZt-J9mz7Y-agNhy4-og4rVR-3fc4Z4-9Wkipc-uX5GDz-53NZVs-izNuX-h5ARaZ-eyR8Rd-GHbqBm-wvkFJB-uqCmQ6-22w5BL-6KcyPa-a6XfaJ-6kyyoj-f6bZno-htZ5VG-eG7sG2-BYQa-nVLCds-oGecaF-wLSaaz-6Y5ejz-9wsL2j-wFonXE-9MppuP-ahZNGv-oj2JbD-NchkTb-a6j7Jz-xfYrgE-dPS2M-8ZeV4V-73Notp-9YBCkP-gtqvjg
+  license: Public Domain
+  creator: Keith Ewing
+  alt: Bee on Purple Coneflower
+source:
+  url: http://mnzoo.org/conservation/act-wildlife/plant-pollinators/
+  name: Minnesota Zoo
+category: habitat
+---
+Purple coneflowers are brightly colored and attract many pollinators.
```

Comment:
I wonder if we could write a little more for this.

Reply:
@floogulinc - Can you make a suggestion? Maybe "Purple coneflowers are brightly colored, are native to Minnesota, and attract many pollinators." since that adds a little more from the source?